makefile

```diff
@@ -1869,6 +1878,60 @@ ifneq (,$(BESM6_BUILD))
     endif
 endif
 
+PDP6D = PDP10

```

Comment:
= PDP6

Reply:
No.  The symbols ending with D are directory names.  All this new KA10, KI10 and PDP6 stuff are in the PDP10 directory with the original KS10.